client/src/service/index.js

```diff
@@ -1,3 +1,3 @@
-export const getApiUrl = () => process.env.REACT_APP_API_URL;
+export const getApiUrl = () => 'spoonacular-recipe-food-nutrition-v1.p.rapidapi.com';
 
-export const getApiKey = () => process.env.REACT_APP_API_KEY;
\ No newline at end of file
+export const getApiKey = () => '###';
```

Comment:
LOL gave up on setting up the env var eh ;)

Reply:
Yup lol